evolver/base.py

```diff
@@ -8,6 +8,7 @@
 import pydantic
 import pydantic_core
 import yaml
+from pydantic._internal._validators import import_string
```

Comment:
Should we be using this? the `_internal` and lack of docs make me think not. It looks like this is used to import from fqcn to a type, perhaps we could either make a util using importlib or a simple wrapper around a pydantic model that has ImportString type?